src/server.js

```diff
@@ -53,9 +54,14 @@ module.exports = function() {
 			log.error("Path to SSL certificate is invalid. Stopping server...");
 			process.exit();
 		}
+		if (config.https.ca.length && !fs.existsSync(caPath)) {
```

Comment:
Those would be changed as well (not as part of your pr). You created the variable, so it should be used here. It's confusing not using it because it leads one to wonder why you aren't using it, and the only reason is because that's what we do with the others.